### ISSUE TITLE
Closing Gide should close open debug variable windows

### ISSUE THREAD

**Issue by rohrlich** [goki/gide#162]

**Comment by rohrlich** [comment]
I noticed that prefs windows aren't closed either so the problem is not specific to debug.

**Comment by rcoreilly** [comment]
Yeah we don't really track "main window" vs. "other windows" wrt closing -- need to add some logic to check if when window closes it is the last "main window" and then it should close all the others.

**Comment by rcoreilly** [comment]
If you just want to quit completely, you should use Quit, not close window.

But I think the key issue here is that you want to close all the "associated" windows for a given main window, while keeping other gide project windows open.

To do that we need to have some way of tracking the original parent through the window creation process which is definitely not in place currently.

**Comment by rohrlich** [comment]
Yes, I think that analysis is good. I often have multiple projects open so having the associated close when I close the main window would be just right.
